traitsui/qt4/button_editor.py

```diff
@@ -149,13 +149,14 @@ def init(self, parent):
         # FIXME: We ignore orientation, width_padding and height_padding.
 
         factory = self.factory
-
+        label  = factory.label or self.item.get_label(self.ui)
```

Comment:
I think we'd prefer `label = factory.label if factory.label != "" else self.item.get_label(self.ui)`  (also flake8 on the number of spaces).